crates/forge/src/shared_cache.rs

```diff
@@ -1,57 +1,57 @@
-pub const CACHE_DIR: &str = ".snfoundry_cache";
-pub const PREV_TESTS_FAILED: &str = ".prev_tests_failed";
-
-use anyhow::{Ok, Result};
+use anyhow::Result;
 use camino::Utf8PathBuf;
 use forge_runner::test_case_summary::AnyTestCaseSummary;
-use scarb_api::{metadata::MetadataCommandExt, ScarbCommand};
 use std::fs::{self, File};
-use std::io::{BufRead, BufReader, BufWriter, Write};
+use std::io::{BufRead, BufReader, BufWriter, ErrorKind, Write};
+
+#[derive(Debug, PartialEq, Default, Clone)]
+pub struct FailedTestsCache {
+    cache_dir: Utf8PathBuf,
+}
+
+const PREV_TESTS_FAILED: &str = ".prev_tests_failed";
```

Comment:
I cannot tell that it is a filename from that var name

Reply:
only moved it, can change name if you have better idea

Reply:
`FILE_WITH_PREV_TESTS_FAILED`